src/sentry/migrations/0878_backfill_open_periods.py

```diff
@@ -27,9 +27,22 @@
 
 # copied constants and enums
 class ActivityType(Enum):
-    SET_RESOLVED = 1
     SET_UNRESOLVED = 2
     SET_REGRESSION = 6
+    SET_RESOLVED = 1
+    SET_RESOLVED_IN_RELEASE = 13
+    SET_RESOLVED_BY_AGE = 15
+    SET_RESOLVED_IN_COMMIT = 16
+    SET_RESOLVED_IN_PULL_REQUEST = 21
+
+
+RESOLVED_ACTIVITY_TYPES = [
+    ActivityType.SET_RESOLVED.value,
+    ActivityType.SET_RESOLVED_IN_RELEASE.value,
+    ActivityType.SET_RESOLVED_BY_AGE.value,
+    ActivityType.SET_RESOLVED_IN_COMMIT.value,
+    ActivityType.SET_RESOLVED_IN_PULL_REQUEST.value,
```

Comment:
nit: if you make `ActivityType` an `IntEnum`, then you can remove the `.value` from each of these :)

Reply:
just copying it from `src/sentry/types/activity.py`